content/docs/command-reference/exp/apply.md

```diff
@@ -0,0 +1,138 @@
+# exp apply
+
+Apply the changes from a previously run experiment into the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc exp apply [-h] [-q | -v] experiment
+
+positional arguments:
+  experiment     Experiment to be applied.
```

Comment:
Core question: Should `experiment` argument be `target` instead for consistency with existing commands?

Reply:
`experiment` here is an experiment name or git SHA, equivalent to a git branch/tag name or git SHA. Not sure what the ideal usage would be here, but `target` seems like it might be confusing with the `dvc repro`/`exp run` `target`?

Reply:
Currently each command gives a different meaning to `target(s)` anyway. It varies a lot, it can be files, stages, .dvc files, dvc.yaml files, directories... (e.g. add, check-ignore, commit, freeze, repro)...

Reply:
But yeah I guess that's not ideal to begin with. And they do tend to be DVC-tracked stuff so I take this back. Resolving

Reply:
If anything I'd consider shortening the arg name to `exp`.